### ISSUE TITLE
Consult the tiny imagenet E100 for data set release

### ISSUE THREAD

**Issue by Junjie31** [VILA-Lab/SRe2L#4]
Your work is very impressive. After checking the data set you released, the data about tiny imagenet is only E50, tiny_rn18E50_[1K].Aug.zip. Would you like to release the data set about tiny imagenet  E100? Thank you very much!

**Comment by zeyuanyin** [comment]
Thank you for your attention to our work. In our study, `rn18E50` represents the recovery (squeezed) model trained for 50 epochs (squeezing budgets). BTW, it's just a model file name.

In Table 2 of our paper, we conducted ablation experiments to explore the effect of squeezing budgets on the final validation accuracy. For the dataset recovered from `rn18E100`, which you requested, it does not yield the best results in our study.

<img width="581" alt="image" src="https://github.com/VILA-Lab/SRe2L/assets/51396847/843fa737-e90a-46c0-a8cd-3c3c0ce1dd4d">


For the squeezed model on Tiny-ImageNet, we modified the official torchvision classification code available at https://github.com/pytorch/vision/tree/main/references/classification to train the ResNet model on Tiny-ImageNet.

**Comment by Junjie31** [comment]
I hope this message finds you well. I wanted to apologize for any misunderstanding regarding the reference to "E." I had initially assumed that "E" might be an abbreviation for an IPC code. However, I now understand that it was a misinterpretation on my part.

I have previously reviewed the provided dataset download link, and it seems that there is access to the Tiny ImageNet IPC50 dataset. Is there a direct download link  for the Tiny ImageNet IPC100 dataset? Thank you!

**Comment by zeyuanyin** [comment]
> I hope this message finds you well. I wanted to apologize for any misunderstanding regarding the reference to "E." I had initially assumed that "E" might be an abbreviation for an IPC code. However, I now understand that it was a misinterpretation on my part.
> 
> I have previously reviewed the provided dataset download link, and it seems that there is access to the Tiny ImageNet IPC50 dataset. Is there a direct download link for the Tiny ImageNet IPC100 dataset? Thank you!

The download link for distilled Tiny ImageNet with IPC100 is  https://huggingface.co/datasets/zeyuanyin/SRe2L/resolve/main/sre2l_tiny_rn18E50_4K_ipc100.zip
